examples/keyfiguresfinland.py

```diff
@@ -0,0 +1,131 @@
+
+# Drop since not needed.
+df.drop('WHOLE COUNTRY', axis = 0, inplace = True)
+
+# The json file used for the mapbox viz.
+# https://geo.stat.fi/geoserver/wfs?service=WFS&version=2.0.0&request=GetFeature&typeName=maakunta1000k_2021&outputFormat=json
+with open('assets/regions.json', encoding = 'utf-8') as f:
+    regions_json = json.loads(f.read())
+
+external_stylesheets = [dbc.themes.SPACELAB]
+
+app = Dash(name = __name__, external_stylesheets = external_stylesheets)
+
+app.layout = dbc.Container([
+        
+        html.Div("Finland's Regional Key Figures", className="text-center mb-3 mt-3 fw-bold display-1"),
+
+        dbc.Row([
+            
+            dbc.Col([
+                html.H2('Key figure'),
+                dcc.Dropdown(id = 'key-figures-finland-key-figure-selection-x',
+                             options = sorted(list(pd.unique(df.columns))),
+                             value = "Degree of urbanisation, %, 2020",
+                             className = 'text-nowrap'
+                             ),                
+                html.H2(id = 'key-figures-finland-whole-country-header-x', className="mt-5 display-2 text-center"),
+                html.Div(['Data by ',html.A('Statistics Finland', href = 'https://pxdata.stat.fi/PxWeb/pxweb/en/Kuntien_avainluvut/Kuntien_avainluvut__2021/kuntien_avainluvut_2021_viimeisin.px/', target = '_blank')], className="text-center fs-3 text"),
+                
+                ], xs = 12, sm = 12, md = 6, lg = 6, xl = 6, xxl = 6, align = 'center'),
+                
+            dbc.Col([
+                dcc.Graph(id = 'key-figures-finland-region-map-x', figure = px.choropleth_mapbox(center = {"lat": 64.961093, "lon": 25.795386}))
+                
+                ], xs = 12, sm = 12, md = 6, lg = 6, xl = 6, xxl = 6)
+
+            ], justify = 'center'),
+  
+        dcc.Store(id = 'key-figures-finland-locations-x'),
+        dcc.Store(id = 'key-figures-finland-zs-x'),
+        dcc.Store(id = 'key-figures-finland-geojson-data', data = regions_json),
+        ], fluid = True)    
+
+@app.callback(Output('key-figures-finland-whole-country-header-x','children'),Input('key-figures-finland-key-figure-selection-x', 'value'))
+def update_whole_country_header(key_figure):
+    
+    # Get all the header components for the header.
+    dff = whole_country_df.loc[key_figure]
+    stat_name = dff.stat_name
+    stat_unit = dff.unit
+    stat_year = dff.year
+    stat_value = dff.value
+ 
+    # Change values with no decimals (.0) to int.
+    stat_value = {True: int(stat_value), False: stat_value}['.0' in str(stat_value)]
+    # Use space as thousand separator.
+    stat_value = "{:,}".format(stat_value).replace(',',' ')
+    
+    return html.Div(
+        [
+            html.Div([stat_name, ", ", stat_year]),
+            html.Div("in Finland"),
+            html.Span([stat_value, " ", stat_unit]),
+        ]
+    )
```

Comment:
This could be made a lot more concise:

```suggestion
 dff = whole_country_df.loc[key_figure]

    # Change values with no decimals (.0) to int.
    stat_value = {True: int(dff.value), False: dff.value}['.0' in str(dff.value)]
    # Use space as thousand separator.
    stat_value = "{:,}".format(stat_value).replace(',', ' ')

    return html.Div(
        [
            html.Div([dff.stat_name, ", ", dff.year]),
            html.Div("in Finland"),
            html.Span([stat_value, " ", dff.unit]),
        ],
    )
```

Reply:
Thanks for the comments, I'll added the fixes on the newly renamed file